MovieDB/MovieDB/AppDelegate.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AppDelegate.swift
+//  MovieDB
+//
+//  Created by tran.duc.tan on 5/14/19.
+//  Copyright © 2019 tranductanb. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+final class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+    var assembler: Assembler = DefaultAssembler()
+    
+    func applicationDidFinishLaunching(_ application: UIApplication) {
+        if NSClassFromString("XCTest") != nil {
+            window?.rootViewController = UnitTestViewController()
+        } else {
+            bindViewModel()
+        }
+    }
+    
+    private func bindViewModel() {
+        guard let window = window else {
```

Comment:
```suggestion
        guard let window = window else { return }
```
ngắn thì viết luôn 1 line anh nhé